db/migrate/20220428135032_rename_id_to_show_id.rb

```diff
@@ -0,0 +1,5 @@
+class RenameIdToShowId < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :shows, :id, :show_id
```

Comment:
we should postpone database stuff until issue 4. Also an fyi `id` by default is integer field, while show_id is a string, looking at the csv file. You should have keep both id and show_id. show_id can be represented as an external ID.